src/commands/init.js

```diff
@@ -248,22 +259,21 @@ function interactiveMenu(directory, list) {
     });
 }
 
-function checkFolder(force = false, directoryName = '') {
+function checkFolder(force = false, directory = '') {
     return new Promise((resolve) => {
-        const directoryPath = path.join(process.cwd(), directoryName);
+        const directoryPath = getAbsolutePath(directory || process.cwd());
```

Comment:
Yepp I had updated my test, but didn't push it, since I wanted your view on things. About case 4 and 6, isn't your example case 6 exactly?

Maybe I should push the latest code including the `getAbsolutePath(path.join(directory, directoryName));` and we do the `process.cwd()` replacing in a separate branch?